src/actions/guides/tutorial/00_overview.cr

```diff
@@ -0,0 +1,163 @@
+
+    ```
+    API only or full support for HTML and Webpack? (api/full): full
+    ```
+
+    Lastly, the wizard would like to know if we want to generate authentication. We do! This will give us the ability
+    to create user accounts and login/out of our accounts. Enter `y`
+
+    ```
+    Generate authentication? (y/n): y
+    ```
+
+    Once that's done, Lucky will give you a few instructions on what to do next. We will follow those instructions
+    in the next step.
+
+    > For more information on the wizard, read the [Starting a Lucky Project](#{Guides::GettingStarted::StartingProject.path}) guide.
+
+    ## Booting Our Application
+
+    Before we boot our application, we still need to complete the setup process. This portion is specific to the
+    individual as only you will know your own databasae settings.
+
+    Let's start by changing the directory in to the project. Enter `cd clover`
+
+    ```bash
+    cd clover
+    ```
+
+    Now we need to make sure we can connect to Postgres so we can run SQL. Open up your Clover app project, and
+    open the `config/database.cr` file.
+
+    The default postgres connection uses the username `postgres`, and the password `postgres` along with the hostname `localhost`.
+    Look over these settings, and update them to match your own personal setup.
+
+    Done? Did you remember to save the file? 😄
+
+    > For more information on setting up your database, read the [Database Setup](#{Guides::Database::DatabaseSetup.path}) guide.
+
+    ### Setup Script
+
+    Next we will run our setup script. This script does a few things:
+
+    * Run a system check for 3rd party dependencies needed to run this app.
+    * Setup our assets (css, javascript) using Yarn
+    * Install Crystal shard dependencies. (e.g. Lucky, Avram, etc...)
+    * Create our database. In this case, a database named `clover_developement`
+    * Verify that we can connect to this new database
+    * Run SQL code (migration) to create our users table. (Added by saying `y` to generate Authentication).
+    * Import sample data in to our database. On first run, there's no sample data.
+
+    Run the setup script. Enter `./script/setup`
+
+    ```bash
+    ./script/setup
+    ```
+
+    > This may take a bit of time to run. If anything fails at any point, let us know!
+
+    ### Congratulations!
+
+    If you see the output `Run 'lucky dev' to start the app`, then
+    you've officially started your first Lucky project! Now you can run the app, and play with it.
+
+    Start the Lucky app. Enter `lucky dev`.
+
+    ```bash
+    lucky dev
+    ```
+
+    At any point you'd like to stop the server, use the `ctrl-C` keyboard command.
+
+    > There's two separate processes that boot; `assets` and `web`. We must wait for both to finish.
+
+    ## Browsing your Application
+
+    Open up your favorite browser, and head over to `http://127.0.0.1:3001`. You should see the Lucky home page.
```

Comment:
When you boot a fresh app, the URL shown in the big blue box is actually `http://127.0.0.1:3001`. I wanted to correlate that because it's kind of confusing when it prints out both 3001 and 5000.
<img width="760" alt="Screen Shot 2021-05-07 at 9 51 04 AM" src="https://user-images.githubusercontent.com/2391/117482812-c2ab9500-af19-11eb-9500-70d31a4e33e2.png">